### ISSUE TITLE
Purpose masks

### ISSUE THREAD

**Issue by Shiro-LK** [deep-diver/EN-FR-MLT-tensorflow#5]
Hello,

Thanks for your repository, it helps a lot !
I am trying to understand the purpose of the masks in your code. I assume the goal is to remove the padding from the loss function, but looking at the get_batches function, it seems that the array of lenghts returned has always the same value as you compute the lengths of the target pad sequence instead of the target sequence. Is it a mistake ? Or the masks has another goal ?
